python/cudf_polars/pyproject.toml

```diff
@@ -52,6 +52,13 @@ version = {file = "cudf_polars/VERSION"}
 [tool.pytest.ini_options]
 xfail_strict = true
 
+[tool.coverage.report]
+exclude_also = [
+  "if TYPE_CHECKING:",
```

Comment:
It's unfortunate that we have to manually exclude this from coverage. It would be nice if the coverage tool were able to exclude it automatically.

Reply:
Yeah